test-e2e/Cargo.toml

```diff
@@ -25,6 +25,11 @@ num-bigint = { version = "0.4.0" }
 halo2-fibonacci = { git = "https://github.com/ElusAegis/halo2-fibonacci-sample.git" }
 halo2-keccak-256 = { git = "https://github.com/ElusAegis/halo2-keccak-stable.git" }
 
+# Nova Scotia dependencies
+nova-scotia = { version = "0.5.0" }
+nova-snark = { version="0.23.0" }
+serde_json = { version="1.0.85" }
+
```

Comment:
we should remove this (for now)
since we don't activate the `nova-scotia` feature
(and it needs to work without these dependencies)

but once you update the type for `udl` (serialization)
it should be solved easily
it can be done in another PR